src/encoded/commands/load_data.py

```diff
@@ -43,8 +43,8 @@ def main():
     load_test_data = DottedNameResolver().resolve(load_test_data)
 
     # do not run on webprod/webprod2 unless we set --prod flag
-    if 'webprod' in env and not args.prod:
-        log.info('load_data: skipping, since we are on webprod/webprod2 and --prod not used')
+    if ('webprod' in env or 'blue' in env or 'green' in env) and not args.prod:
```

Comment:
I might factor out `PRODUCTION_NAME_PARTS = ('webprod', 'blue, 'green')` and then do
```
if any(part in env for part in PRODUCTION_NAME_PARTS):
```

Reply:
Although in `create_mapping_on_deploy.py` you actually have variables that know the full name of these, so it might be good to rationalize how this is determined. But I'd bargain down to just a TODO to look at this later. Really what is here is fine.